### PR TITLE
Add CC BY-SA 4.0 license

### LINKED ISSUE #257: Suggestion to add a license file to the repository
Hello! I came across your code and I was wondering if you have considered adding a license file to your repository. A license file is a way of letting others know how they can use or modify your code legally. Without a license file, your code is technically not open source and no one can use it without your permission. This might limit the potential impact and reach of your project. GitHub recommends that you include a license in your repository to let others know how can your code be used and built upon (Source: [Licensing a repository](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository)).  If you are interested in making your project open source, you can check out [Choosing a license](https://choosealicense.com/no-permission/) to learn more about the different types of licenses and their benefits. 

To add a license file to your repository, you can follow the steps in this guide: [Adding a license to a repository](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also read this article for more information on open source licensing: [Open Source Licensing Guide](https://github.com/readme/guides/open-source-licensing).

### PR DESCRIPTION
**Pull request by scordio**
Closes #257.

### CODE REVIEW

File: generator/src/main/java/org/javabubble/generator/site/ReadmeHTML.java [hunk 1]
```diff
@@ -127,22 +128,32 @@ private DomContent person(JavaPerson person) {
...
 	}
 
+	private DomContent copyright() {
+		return p( //
+				text("This work is licensed under "), //
+				a("CC BY-SA 4.0") //
+						.withHref("http://creativecommons.org/licenses/by-sa/4.0/") //
+						.withTarget("_blank") //
+						.withRel("license noopener noreferrer") //
+						.withStyle("display:inline-block;"));
```
Comment (marchof) [inline generator/src/main/java/org/javabubble/generator/site/ReadmeHTML.java:156]: What is the purpose of the style definition? We don't have it for the other links and I can't see any difference. Sorry, I'm not in web development at all.

File: generator/src/main/java/org/javabubble/generator/site/ReadmeHTML.java [hunk 2]
```diff
@@ -127,22 +128,32 @@ private DomContent person(JavaPerson person) {
...
+	}
```
Comment (marchof) [inline generator/src/main/java/org/javabubble/generator/site/ReadmeHTML.java:157]: Add a period (`.`) at the end of the sentence?

File: generator/src/main/java/org/javabubble/generator/site/ReadmeHTML.java [hunk 3]
```diff
@@ -127,22 +128,32 @@ private DomContent person(JavaPerson person) {
...
+	private DomContent handleWithLink(Handle handle, SocialIcons icon, String title) {
 		if (handle != null) {
 			return td(a(icon.svg()).withTitle(title + ": " + handle.getHandle()).withHref(handle.getWebLink()));
 		} else {
 			return td();
 		}
 	}
 
+	private DomContent copyright() {
+		return p( //
```
Comment (marchof) [inline generator/src/main/java/org/javabubble/generator/site/ReadmeHTML.java:150]: I would add a header (h2) "License" here.

**Review by marchof — CHANGES_REQUESTED** [review]
Thanks for taking care of this! I left some minor formal proposals.

File: generator/src/main/java/org/javabubble/generator/site/ReadmeHTML.java (hunk 1 above)
Comment (scordio) [reply generator/src/main/java/org/javabubble/generator/site/ReadmeHTML.java:156]: I'm not into web development either 😅

Honestly, I've taken inspiration from https://chooser-beta.creativecommons.org/ but I also don't feel an advantage. I'll get rid of it.

File: generator/src/main/java/org/javabubble/generator/site/ReadmeHTML.java [hunk 4]
```diff
@@ -127,22 +128,34 @@ private DomContent person(JavaPerson person) {
...
 	}
 
+	private DomContent copyright() {
+		return each( //
+				h2("License"), //
+				p( //
+					text("This work is licensed under "), //
+					a("CC BY-SA 4.0") //
+							.withHref("http://creativecommons.org/licenses/by-sa/4.0/") //
+							.withTarget("_blank") //
```
Comment (marchof) [inline generator/src/main/java/org/javabubble/generator/site/ReadmeHTML.java:156]: In this case I would also prefer to remove those directive: I personally hate web sites which try to explain me how to use hyperlinks.

**Review by marchof — APPROVED** [review]

**Comment by marchof** [comment]
Thanks!

**Pull request merged**